tools/clang/test/DXC/hlsl_ver.test

```diff
@@ -0,0 +1,3 @@
+// HLSL Version
+// RUN: %dxc /T ps_6_0 %S/Inputs/smoke.hlsl  /HV 2016
+// RUN: not %dxc /T ps_6_0 %S/Inputs/smoke.hlsl  /HV 2015
```

Comment:
If we merge the not line for this into the other HLSL version test, we could just delete this test. IMO the other test is more robust and tests all cases.